tests/unit/inference/test_inference.py

```diff
@@ -213,6 +215,17 @@ def assert_fn(model_w_task):
         NotImplementedError(f'assert_fn for task "{task}" is not implemented')
     return assert_fn
 
+@pytest.fixture
+def check_injection(model):
+    def verify_injection(module):
+        for child in module.children():
+            if isinstance(child, nn.ModuleList):
+                assert isinstance(child[0], DeepSpeedTransformerInference),
```

Comment:
here seems to be syntax mistake. I think you would need `\` at the end of the line.

Reply:
yes, corrected now